README.md

```diff
@@ -730,6 +730,8 @@ In order to achieve this kind of workflow, all you need to do is specify the dep
 codePush.sync({ deploymentKey: userProfile.CODEPUSH_KEY });
 ```
 
+*NOTE: Deployment key specified by `codePush.sync({ deploymentKey: key })` will not persist across app restarts (user kills app process and launch it again) and will be taken from native-side on next app launch. In case app restarts programmatically (while using IMMEDIATE installation mode) deployment key will persists.*
```

Comment:
I'm not sure what is meant by persisting after the app restarts programatically. As far as I understood, the key only persists for the duration of the call, regardless of when the call is made, so:

```
codePush.sync({ deploymentKey: key })   // This call uses the specified key, regardless of when it is called
    .then(() => {
        codePush.sync();    // This call uses the key specified natively, regardless of when it is called
    });
```

Have I gotten something confused here? If I haven't gotten it muddled up, then maybe this is what you mean?

```
*NOTE: The deployment key specified via `codePush.sync({ deploymentKey: key })` does not persist beyond the given call, and needs to be specified as an option each time it is used.*
```

I'm not sure that using `IMMEDIATE` install mode or not makes any difference - are you saying that the value set to the variable `key` is lost when the app restarts, but not when it restarts programmatically? Because that's not what I'd expect.